src/api/cvc4cpp.cpp

```diff
@@ -945,13 +946,26 @@ std::ostream& operator<<(std::ostream& out, const Result& r)
 /* -------------------------------------------------------------------------- */
 
 Sort::Sort(const Solver* slv, const CVC4::Type& t)
```

Comment:
Shouldn't this argument be a `TypeNode`?

Reply:
There are current interfaces for both Type and TypeNode, due to the issue I mentioned in the previous comment.  (This is the same situation we are currently in with Term which has constructors for Expr/Node).

This will take a few more PRs to eliminate.